### DISCUSSION TITLE
File Downlaod TypeError: initialData is null

### DISCUSSION THREAD

**Discussion by arsibux** [livewire/livewire#5069] [1. Help]
**Download File**
When I click button having file download function 
public function exportPdf(){
  return response()->download($this->pdfPath);
}
Face this error TypeError: initialData is null
 and struck every thing and need to page load.

**Comment by joshhanley** [comment]
@arsibux that's strange! Can you share your component class and view? And also how it is rendered on the page (ie is it a full page component or rendered in a view from a controller, etc)?

Thanks!
